sdcm/nemesis.py

```diff
@@ -2818,16 +2818,15 @@ def execute_data_validation_thread(command_template, keyspace_name, number_of_ro
             self.log.info("Restoring the schema of the keyspace '%s'", chosen_snapshot_info["keyspace_name"])
             restore_task = mgr_cluster.create_restore_task(restore_schema=True, location_list=location_list,
                                                            snapshot_tag=chosen_snapshot_tag)
-            step = 10
-            extra_time = 300
-            restore_task.wait_and_get_final_status(step=step,
-                                                   timeout=chosen_snapshot_info['expected_timeout'] / step + extra_time)
+
+            restore_task.wait_and_get_final_status(step=10, timeout=chosen_snapshot_info['expected_timeout'])
             assert restore_task.status == TaskStatus.DONE, f'Schema restoration of {chosen_snapshot_tag} has failed!'
             self.cluster.restart_scylla()  # After schema restoration, you should restart the nodes
 
         restore_task = mgr_cluster.create_restore_task(restore_data=True,
                                                        location_list=location_list, snapshot_tag=chosen_snapshot_tag)
-        restore_task.wait_and_get_final_status(step=30, timeout=chosen_snapshot_info["expected_timeout"])
+        timeout_plus_extra_time = 1.3 * chosen_snapshot_info['expected_timeout']
```

Comment:
where is 1.3 coming from ?

Reply:
i'm giving it a tolerance to timeout to another 30%

Reply:
1 out of 4 nemesis runs passed [here](https://argus.scylladb.com/test/37bfd989-07ba-4778-8899-36d02a013aa9/runs?additionalRuns[]=fb4c19af-675c-4717-b0a2-346fac44c5b6)

still need some work, or to cancel in this case nemesis during prepare (will test that too)

Reply:
> i'm giving it a tolerance to timeout to another 30%

If we need extra time, why not update the source yaml with this extra time ?